Sources/Extensions/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -52,7 +52,8 @@ public extension Collection {
 public extension Collection where Index == Int, IndexDistance == Int {
 	
 	/// SwifterSwift: Random item from array.
-	public var randomItem: Iterator.Element {
+	public var randomItem: Iterator.Element? {
+        guard !isEmpty else { return nil }
```

Comment:
Could you please remove a space before **guard**

Reply:
hey @akuzminskyi Sorry I don't get it 😅 What space?

Reply:
The line is a git addition so it looks offset from the `+`

Reply:
@LucianoPAlmeida Do we still need the `Iterator` prefix on `Element`?

Reply:
@SD10 No, I think we can remove it :)